src/components/CodeReminderModal/index.jsx

```diff
@@ -214,35 +214,33 @@ export class BaseCodeReminderModal extends React.Component {
     } = this.state;
 
     return (
-      <>
-        <Modal
-          ref={this.modalRef}
-          dialogClassName="code-reminder"
-          title={this.renderTitle()}
-          body={this.renderBody()}
-          buttons={[
-            <Button
-              key="remind-submit-btn"
-              disabled={submitting}
-              className="code-remind-save-btn"
-              onClick={handleSubmit(this.handleModalSubmit)}
-            >
-              <>
-                {mode === REMIND_MODE && submitting && <Icon className="fa fa-spinner fa-spin mr-2" />}
-                Remind
-              </>
-            </Button>,
-            <SaveTemplateButton
-              key="save-remind-template-btn"
-              templateType={REMIND_MODE}
-              setMode={this.setMode}
-              handleSubmit={handleSubmit}
-            />,
-          ]}
-          onClose={onClose}
-          open
-        />
-      </>
+      <Modal
+        ref={this.modalRef}
+        dialogClassName="code-reminder"
+        title={this.renderTitle()}
+        body={this.renderBody()}
+        buttons={[
+          <Button
+            key="remind-submit-btn"
+            disabled={submitting}
+            className="code-remind-save-btn"
+            onClick={handleSubmit(this.handleModalSubmit)}
+          >
+            <>
```

Comment:
Are we trying to get rid of all of these shorthand containers? Or just the ones that wrap an entire component?

Reply:
@kiram15 There's definitely still valid use cases for the React Fragments (`<></>`), like we see here on this line. We want to keep using it here because we want to use the `{...}` as a sibling to the "Remind" text, without getting wrapped by another DOM element (e.g., `div`).

In the case of the fragment that was removed, `Modal` was the only child element of the top-level fragment in this component so the fragment was essentially irrelevant/useless. 

The only reason to use a fragment at the top-level of a component is if the component has multiple sibling nodes at the top-level, but you don't want to include a wrapper `div` (or other element), e.g.:

```jsx
<>
  <Button>Hello</Button>
  <Button>World</Button>
</>
```

Reply:
@kiram15 I take that back actually. I think you're right that this Fragment could actually be removed, too (I had expected it to throw an ESLint error when I removed it just now, but it didn't!)